### DISCUSSION TITLE
e91593857c82e4fc56f46aded9819473

### DISCUSSION THREAD

**Discussion by giscus** [Meiting-Wang/Meiting-Wang.github.io#9] [Announcements]
# e91593857c82e4fc56f46aded9819473

我们前面通过文章利用 Github Pages + jekyll-TeXt-theme 初步搭建个人博客介绍了初步搭建博客的流程。在之后的使用中，发现其提供的 comments（disqus、gitalk、valine）有些问题，经过网上查询和个人实践，发现基于 github issues 的 utteranc ...

https://meiting-wang.github.io/e91593857c82e4fc56f46aded9819473

**Comment by Meiting-Wang** [comment]
Giscus, an awsome comment system, can be deployed in [giscus app](https://giscus.app/zh-CN).

**Reply by Meiting-Wang** [reply]
We can reply here, like github's discussions.
